python/ql/src/experimental/semmle/python/frameworks/Stdlib.qll

```diff
@@ -8,4 +8,234 @@ private import semmle.python.dataflow.new.DataFlow
 private import semmle.python.dataflow.new.TaintTracking
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import experimental.semmle.python.Concepts
-private import semmle.python.ApiGraphs
+
+/** Provides models for the Python standard library. */
+private module Stdlib {
+  // ---------------------------------------------------------------------------
+  // re
+  // ---------------------------------------------------------------------------
+  private module Re {
+
+    /** Gets a reference to the `re` module. */
+    private DataFlow::Node re(DataFlow::TypeTracker t) {
+      t.start() and
+      result = DataFlow::importNode("re")
+      or
+      exists(DataFlow::TypeTracker t2 | result = re(t2).track(t2, t))
+    }
+
+    /** Gets a reference to the `re` module. */
+    DataFlow::Node re() { result = re(DataFlow::TypeTracker::end()) }
+
+    /**
+     * Gets a reference to the attribute `attr_name` of the `re` module.
+     * WARNING: Only holds for a few predefined attributes.
+     */
+    private DataFlow::Node re_attr(DataFlow::TypeTracker t, string attr_name) {
+      attr_name in ["match", "fullmatch", "search", "split", "findall", "finditer", "sub", "subn", "compile"] and
```

Comment:
Thanks @mrthankyou for the reminder about adding `sub` and `subn` to the list of vulnerable methods. 😄